src/aap_eda/services/activation/activation_manager.py

```diff
@@ -106,6 +109,12 @@ def _increase_restart_count(self):
         self.db_instance.restart_count += 1
         self.db_instance.save(update_fields=["restart_count", "modified_at"])
 
+    @run_with_lock
+    def _reset_log_tracking_id(self, log_tracking_id: str):
```

Comment:
This name is completely misleading because the word "reset" suggests that the tracking id of the activation can change which is not true since we want to keep it forever once it is assigned.